types/office-js/index.d.ts

```diff
@@ -8807,6 +8857,22 @@ declare namespace Office {
          * <tr><td>{@link https://docs.microsoft.com/outlook/add-ins/#extension-points | Applicable Outlook mode}</td><td>Appointment Organizer</td></tr></table>
          */
         optionalAttendees: Recipients;
+        /**
+         * Gets the email address of the organizer for the specified meeting.
+         *
+         * The organizer property returns an Organizer object that provides a method to get the organizer value.
```

Comment:
Consider merging these sentences to provide a clearer picture of what this property is actually doing.